luaui/Widgets/gui_options.lua

```diff
@@ -3209,6 +3217,12 @@ function init()
 		-- INTERFACE
 		{ id = "label_ui_interface", group = "ui", name = Spring.I18N('ui.settings.option.label_interface'), category = types.basic },
 		{ id = "label_ui_interface_spacer", group = "ui", category = types.basic },
+		{ id = "language", group = "ui", category = types.basic, name = Spring.I18N('ui.settings.option.language'), type = "select", options = languageNames, value = languageCodes[Spring.I18N.getLocale()],
```

Comment:
what ensures priority? say, if i set this regular language to FR and the dev landuage to DE.
* looks like right now the last option touched takes precedence. this is somewhat bad but maybe acceptable since randoms shouldnt touch the dev ones. 
* what if i restart the game? are settings saved in a way that makes this robust?

Reply:
- Correct, whatever is touched last becomes the current language. There is no conflict between the two, if you choose something on public it becomes the selected entry in dev, and conversely if you choose something on dev it becomes the selected entry on public (the caveat here is that you choose an entry that's not available on public, the selected value appears blank on the public dropdown, which IMO is fine as it's only for devs).
- This is the exact same as what we have right now if you change your language in a dev version and then switch to the non-dev version: the game stays in the language selected, even if it's not a setting available to the public. E.g. right now if you choose Deutsch in dev, switch to non-dev, your game is still in Deutsch and you don't have any option to change that (outside of going back to dev or manually editing your config file). With this change in, if you choose Deutsch, the same thing happens, except that now you are allowed to switch back to English or French.